src/bin/diff.js

```diff
@@ -2,6 +2,10 @@
 
 'use strict'
 
+var size = require('window-size')
+var availableWidth = size.width || /* istanbul ignore next */ 80
+var ui = require('cliui')({width: availableWidth})
```

Comment:
Available width for tabular display is calculated with respect to individual terminal (window) settings.